100of100Service/src/main/java/dev/coms4156/project/Department.java

```diff
@@ -1,25 +1,64 @@
 package dev.coms4156.project;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * This class represents a department in the organization.
  * Designed under the Composite Design Pattern.
  */
 public class Department extends OrganizationComposite {
   private Employee head;
+  private final List<Employee> employees;
 
   /**
    * Constructs a department with the given ID and name.
-   * @param db the HR database facade that manages the department
-   * @param id the ID of the department
+   *
+   * @param db   the HR database facade that manages the department
+   * @param id   the ID of the department
    * @param name the name of the department
    */
-  public Department(HRDatabaseFacade db, long id, String name) {
+  public Department(HRDatabaseFacade db, int id, String name, List<Employee> employees) {
```

Comment:
This parameter is unused in its constructor.
And, as said, an overloaded constructor might be helpful.